test/test_blockproduct.jl

```diff
@@ -0,0 +1,102 @@
+using BlockArrays, Test
+
+@testset "Khatri Rao Product (size)" begin
+    #A size
+    m = 5
+    n = 6
+
+    #A block size
+    mi = [3, 2]
+    ni = [4, 1, 1]
+    #B size
+    p = 8
+    q = 7
+    
+    #B block size
+    pi = [5, 3]
+    qi = [3, 2, 2]
+
+    @assert sum(mi) == m
+    @assert sum(ni) == n
+    @assert sum(qi) == q
+    @assert sum(pi) == p
+
+    A = BlockArray(ones(m, n), mi, ni)
+    B = BlockArray(ones(p, q), pi, qi)
+
+    AB = khatri_rao(A, B)
+    
+    @test blocksize(AB) == blocksize(A)
+    @test blocksize(AB) == blocksize(B)
+
+    #Test: Size of resulting blocks
+    for i in 1:blocksize(AB)[1]
```

Comment:
I'd write `for i in blockaxes(AB,1)`

Reply:
Do I make the change and I pull again ? It is my first pull request!

Reply:
Just commit and push your changes to the same branch, it will auto-update the PR